test/plugins/scenarios/gozero/pb/userpb/user_grpc.testpb.go

```diff
@@ -0,0 +1,163 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
I think these files can be generated by the plugin test, right?

Reply:
This was generated using the `goctl` command to enable testing of the complete framework tracing.

Reply:
Yes, I mean you can move this generated command to the scenario.sh file to make these files generate automatically.